URL-API.md

```diff
@@ -4,77 +4,289 @@ MazeMap URL API
 This document describes the basic use of the MazeMap URL API.
 
 
-0. Specifying the URL API version
+Specifying a specific URL API version
 --------------------
-The URL API supports versioning using the _**v**_ parameter, however it is not necessary to specify, as the API assumes the latest version by default. There are legacy functionality from version 0.5 that still works. More on that further down.
 
 ```
 http://use.mazemap.com/?v=1
 ```
 
+* `v` defines the version, in this case `1`.
 
-1. Specifying a campus
+The URL API supports versioning using the `v` parameter, however it is not necessary to specify as the API assumes the latest version by default. In the following examples we omit this parameter. There is legacy functionality from version 0.5 that still works. More on that further down.
+
+
+Specifying a campus
 --------------------
-To start mazemap with a specific campus in view, use the _**campusid**_ parameter. For other parameters, you must also specify the campus to provide proper context.
+
+To start MazeMap with a specific campus in view:
 
 ```
 http://use.mazemap.com/?campusid=1
 ```
->#####campusid's#####
->1. NTNU
->3. St.Olavs Hospital
->5. UiT
+
+* `campusid` defines the campus in view, in this case `1`.
+
+To specify other parameters you must also specify the campus to provide proper context.
+
+#### Examples of campusids
+
+* 1 = NTNU Gløshaugen
+* 3 = St. Olavs Hospital
+* 4 = OSL
+* 5 = UiT
+
+A full list of campus IDs migth be available in the future.
 
 
-2. Specifying a POI destination
+Specifying a POI (Point of Interest)
 --------------------
```

Comment:
Do not mix markdown-style headers like

```markdown
# Header
```

with ReStructuredText-style headers like

```markdown
Header
--------------------
```

Pick a style and stick to it.